packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```diff
@@ -136,20 +136,115 @@ install kernel/nvidia-peermem/nvidia-peermem.o %{buildroot}%{_cross_datadir}/nvi
+ln -s libnvoptix.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libnvoptix.so.1
+
+# Graphics GLVND libs
+install -m755 libnvidia-glvkspirv.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+install -m755 libnvidia-cbl.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+
+install -m755 libGLX_nvidia.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+ln -s libGLX_nvidia.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libGLX_nvidia.so.0
+
+install -m755 libEGL_nvidia.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+ln -s libEGL_nvidia.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libEGL_nvidia.so.0
+
+install -m755 libGLESv2_nvidia.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+ln -s libGLESv2_nvidia.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libGLESv2_nvidia.so.2
+
+install -m755 libGLESv1_CM_nvidia.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+ln -s libGLESv1_CM_nvidia.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libGLESv1_CM_nvidia.so.1
+
+# Graphics Compat
+install -m755 libGL.so.1.7.0 %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+ln -s libGL.so.1.7.0 %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libGL.so.1
+
+install -m755 libEGL.so.1.1.0 %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+ln -s libEGL.so.1.1.0 %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libEGL.so.1
+
+install -m755 libGLESv1_CM.so.1.2.0 %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+ln -s libGLESv1_CM.so.1.2.0 %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libGLESv1_CM.so.1
+
+install -m755 libGLESv2.so.2.1.0 %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+ln -s libGLESv2.so.2.1.0 %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libGLESv2.so.2
+
 %if "%{_cross_arch}" == "x86_64"
+# Binaries
 install -m 755 nvidia-ngx-updater %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}
-%endif
 
-# TODO: add remaining libraries
-# misc
-# Add libnvidia-ml.so for testing purposes
-install -m755 libnvidia-ml.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+# Compute libs
+install -m755 libnvidia-compiler.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
```

Comment:
nit: it's a style thing, but I'd personally do these arch conditionals in the same sections of the spec as the other libraries in the category - compute, etc.

Reply:
:smile: I had it as you describe it, but I thought that people wouldn't like all the `if`s in the spec. I'll change it back.